packages/@lwc/template-compiler/src/codegen/formatters/function.ts

```diff
@@ -10,6 +10,7 @@ import { TEMPLATE_FUNCTION_NAME, TEMPLATE_MODULES_PARAMETER } from '../../shared
 
 import CodeGen from '../codegen';
 import { identifierFromComponentName, generateTemplateMetadata } from '../helpers';
+import { optimizeStaticExpressions } from '../optimize';
```

Comment:
If you're reading this in the GitHub UI, you can skip to here because everything above is just snapshot changes.